sources/packages/web/src/services/AuthService.ts

```diff
@@ -24,6 +24,7 @@ export default async function(
     });
     if (keycloak.authenticated) {
       store.dispatch("auth/login", keycloak);
```

Comment:
With the introduction of "setStudentProfileData", do we still need "store.dispatch("auth/login", keycloak)"?